### PR TITLE
Removed enum in sensors.account_balance. Fixes #5

### LINKED ISSUE #5: All currency properties should allow ISO codes from ISO 4217
All currency related properties should allow all ISO codes from [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) [Official Spreadsheet](http://www.currency-iso.org/dam/downloads/lists/list_one.xls)

It only requires adding all the ISO codes to the enum.

This does currently only affect `sensors.account_balance`.

### PR DESCRIPTION
**Pull request by the-metalgamer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ Changes should start with one of the following tags:
 
 ## v0.14 (unreleased)
 
-- ...
+- [changed] Removed `enum` in `sensors.account_balance`
```
Comment (dbrgn) [inline CHANGELOG.md]: Maybe reword this to:

    [changed] The value in `sensors.account_balance` can now be any ISO 4217 string

File: 14-draft.json [hunk 2]
```diff
@@ -894,14 +894,8 @@
...
               "unit": {
-                "description": "What's the currency? If your currency is missing open a new <a href=\"https://github.com/slopjong/OpenSpaceLint/issues\" target=\"_blank\">issue</a> and request the addition of your currency according <a href=\"http://de.wikipedia.org/wiki/ISO_4217\" target=\"_blank\">ISO 4217</a>.",
-                "type": "string",
-                "enum": [
-                  "BTC",
-                  "EUR",
-                  "USD",
-                  "GBP"
-                ]
+                "description": "What's the currency? It should be formated according to <a href=\"http://de.wikipedia.org/wiki/ISO_4217\" target=\"_blank\">ISO 4217</a> short-code format.",
```
Comment (dbrgn) [inline 14-draft.json]: Could you make the link post to the English Wikipedia, and use https?

**Comment by the-metalgamer** [comment]
I've updated to all proposed changes

File: 14-draft.json [hunk 3]
```diff
@@ -894,14 +894,8 @@
...
               "unit": {
-                "description": "What's the currency? If your currency is missing open a new <a href=\"https://github.com/slopjong/OpenSpaceLint/issues\" target=\"_blank\">issue</a> and request the addition of your currency according <a href=\"http://de.wikipedia.org/wiki/ISO_4217\" target=\"_blank\">ISO 4217</a>.",
-                "type": "string",
-                "enum": [
-                  "BTC",
-                  "EUR",
-                  "USD",
-                  "GBP"
-                ]
+                "description": "What's the currency? It should be formated according to <a href=\"https://en.wikipedia.org/wiki/ISO_4217\" target=\"_blank\">ISO 4217</a> short-code format.",
```
Comment (dbrgn) [inline 14-draft.json]: Found another typo, `formated` should be `formatted` :)

**Comment by the-metalgamer** [comment]
Done all the change requests!

**Comment by dbrgn** [comment]
Thank you!
